eng/common/scripts/Helpers/DevOps-WorkItem-Helpers.ps1

```diff
@@ -513,6 +526,7 @@ function CreateOrUpdatePackageWorkItem($lang, $pkg, $verMajorMinor, $existingIte
     return
   }
   $pkgName = $pkg.Package
+  $pkgGroupId = if ($pkg.PSObject.Properties.Name -contains "GroupId") { $pkg.GroupId } else { $pkg.Group }
```

Comment:
Are we making a bad assumption that if Group is a property if GroupId isn't? Should we check to see if Group exists as well?